source/community-guidelines.md

```diff
@@ -61,7 +61,7 @@ complementary_content: |-
     actively remove someone from the community. Ultimately, we can only use our
     voice to publicly declare that we do not agree with or endorse people or
     organizations who don't act in the spirit of these guidelines.
-complementary_content_aria_label: 'Resources'
+no_container: false
```

Comment:
I don't think this should have any impact? Pretty sure `false` and omitted should be the same.